JamWiFi/ANListView.m

```diff
@@ -134,6 +134,7 @@ - (void)scanInBackground {
         self->interfaceName = interface.interfaceName;
         NSError * error = nil;
         NSArray * nets = [[interface scanForNetworksWithSSID:nil error:&error] allObjects];
+        NSLog(@"Nets%@", nets.firstObject);
```

Comment:
Please remove these logs or format them nicer if you believe they should be kept in master